### ISSUE TITLE
Can't add an external emulator on Android

### ISSUE THREAD

**Issue by Cee123** [zach-morris/plugin.program.iagl#30]
I could't find the following Android emulators under external emulators in the launch settings:
- Irata Jaguar
- Real3DOPlayer

Any chance they could add these in a future update?

I tried to add them several times using the command launch, and using different commands, but no luck.

**Comment by Cee123** [comment]
I tried to do it manually using these commands
/system/bin/am start --user 0 -n ru.vastness.altmer.real3doplayer/ru.vastness.altmer.real3doplayer.MainActivity -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -e %ROM_PATH%

/system/bin/am start --user 0 -n ru.vastness.altmer.iratajaguar/ru.vastness.altmer.iratajaguar.MainActivity -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -e %ROM_PATH%

Still don't know what I'm doing wrong.

**Comment by zach-morris** [comment]
It's possible this is due to the way IAGL currently sends the launch command for android.  The next update will include some options to change this.  As far as I can tell, those commands look correct - there's not a lot on the internet about these two emulators, so I can't be sure, but based on some hyperspin forum posts, those look correct.

**Comment by Cee123** [comment]
Neither of those commands appear to work or do anything. Maybe best to stick with 4do and virtual jaguar even though they're much slower.

**Comment by zach-morris** [comment]
Well after some investigation, it appears this issue is emulator specific.  See [here](https://hyperspin-fe.com/forums/topic/26261-real3doplayer/).  Without the apk being updated by the developer, there's nothing I can do to fix this

**Comment by Cee123** [comment]
Yeah that's alright. No worries. You are right there does seem to be a problem with the emulators themselves. All good.

**Comment by Cee123** [comment]
Well, I managed to have a look at the source code of these emulators using a decompiler and that didn't help. The commands I used themselves seemed to be the correct ones. There's been no updates for either emulator in about two years. I've even emailed the developer from this project. I emailed him twice asking about the command line you'd use to start this emulator. This was last year. And unlike the guys on that forum, I did not receive any response. I'm starting to think these emulators are abandoned altogether, with no release of the source code and no ports to other systems. Which is a shame considering the current state of 3DO and Jaguar emulation on Retroarch.
